src/Home.js

```diff
@@ -36,6 +36,23 @@ const dateCompare = (d1, d2) => {
     }
 }
 
+const SortIcon = ({sort, dir, sortItem, ...props}) => {
+	return (
+		<span aria-hidden="true">
+		{ props.props === sort
+			? <img src="images/sort.svg" className={`${dir === 'asc' ? styles.asc : styles.desc}`} alt="" />
+			: ''
+		}
+		</span>
+
+	);
+};
+
+const SortHeading = ({sort, dir, sortItem, ...props}) => {
+	return (
+		<th aria-sort={sort === props.id ? dir : 'none'} className={props.showmodbile === 'false'? styles.showDesktop : ''} onClick={() => sortItem(props.id)}><button aria-label={props.arialabel} className={styles.clickDisabled} onClick={() => sortItem(props.id)}>{props.name}<SortIcon props={props.id} sort={sort} /></button></th>
```

Comment:
this is me being lazy, but you should prolly not do `...props` and just destructure `showmodbile` and the others